source/js/languageSelector.js

```diff
@@ -0,0 +1,12 @@
+const mobileQuery = getComputedStyle(document.body).getPropertyValue("--phoneWidth");
+const isMobile = () => window.matchMedia(mobileQuery).matches;
+
+if(!isMobile()) {
+  languageSelector = document.getElementsByClassName("language-selector-actual")[0];
```

Comment:
```suggestion
  languageSelector = document.querySelector("language-selector-actual");
```

This also should work ;-)

...but keep in mind previous class name suggestions ;-)